test/AutoRest.TestServer.Tests/Infrastructure/TestServerTestBase.cs

```diff
@@ -71,14 +71,15 @@ public Task TestStatus(Func<string, HttpPipeline, Task<Response>> test)
             Assert.AreEqual(200, response.Status, "Unexpected response " + response.ReasonPhrase);
         });
 
-        public Task Test(Func<string, HttpPipeline, Task> test)
+        public Task Test(Func<string, HttpPipeline, Task> test, bool ignoreScenario = false)
```

Comment:
This is because there are routes in the javascript file that don't have a 'coverage' associated with them. So, this is a workaround to be able to use them, since they are still valid as test cases.